datadog_checks_base/datadog_checks/base/utils/agent/utils.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def should_profile_memory(datadog_agent, check_name):
+    tracemalloc_whitelist = datadog_agent.get_config('tracemalloc_whitelist') or ''
+    if tracemalloc_whitelist:
+        tracemalloc_whitelist = [check.strip() for check in tracemalloc_whitelist.split(',') if check]
```

Comment:
```suggestion
        tracemalloc_whitelist = [check.strip() for check in tracemalloc_whitelist.split(',') if check.strip()]
```

or 

```suggestion
        tracemalloc_whitelist = [check.strip() for check in tracemalloc_whitelist.split(',')]
```

Not sure `if check` is needed here.

Reply:
Need that in case of empty string